src/_DetailBase.js

```diff
@@ -870,6 +982,22 @@ const __class = declare('argos._DetailBase', [View], {
    */
   clear: function clear() {
     this.set('detailContent', this.emptyTemplate.apply(this));
+    if (this.tabList) {
+      domConstruct.empty(this.tabList);
+      if (this.moreTabList) {
+      domConstruct.empty(this.moreTabList);
+      this.moreTabList.style.left = '';
```

Comment:
Use dojo/dom-style to be consistent with other DOM operations that are already using dom-construct, etc.